src/vehicle_gui.cpp

```diff
@@ -2732,7 +2736,7 @@ struct VehicleViewWindow : Window {
 		const Vehicle *v = Vehicle::Get(this->window_number);
 		switch (widget) {
 			case WID_VV_START_STOP:
-				size->height = std::max(size->height, std::max(GetSpriteSize(SPR_FLAG_VEH_STOPPED).height, GetSpriteSize(SPR_FLAG_VEH_RUNNING).height) + WD_IMGBTN_TOP + WD_IMGBTN_BOTTOM);
+				size->height = std::max(size->height, std::max(GetSpriteSize(SPR_WARNING_SIGN).height, std::max(GetSpriteSize(SPR_FLAG_VEH_STOPPED).height, GetSpriteSize(SPR_FLAG_VEH_RUNNING).height)) + WD_IMGBTN_TOP + WD_IMGBTN_BOTTOM);
```

Comment:
`std::max({foo, bar, baz})` using an initializer list would probably be better than chaining std::max calls